cloud_edition/flexibility_mode/docs/crontasks.rst

```diff
@@ -64,7 +64,7 @@ If you don't want to use this wrapper you can prepend `SHELL=/bin/bash`, for exa
     #Ansible: pimee:project:recalculate
     0 2 * * * pimee:project:recalculate --env=prod
     #Ansible: pimee:franklin-insights:fetch-products
-    30 */1 * * * pimee:franklin-insights:fetch-products --env=prod
+    0,30 * * * * pimee:franklin-insights:fetch-products --env=prod
```

Comment:
Why do we go with fixed minutes, all jobs will be launched at the same time on the cluster.. not sure that is really what we want.
Something like that would have been better I think :
`*/30 * * * *`